vnext/ReactUWP/ReactUWP.vcxproj

```diff
@@ -106,7 +109,7 @@
         $(ReactNativeWindowsDir);
         $(ReactNativeWindowsDir)Common;
         $(ReactNativeWindowsDir)JSI\Shared;
-        $(ReactNativeWindowsDir)Pch;
+        $(ReactNativeWindowsDir)ReactUwp\Pch;
```

Comment:
>ReactUwp [](start = 32, length = 8)

Good catch! nit: Please keep the original folder name spelling: ReactUWP.

Reply:
Going to go ahead and merge as is since we will be nuking the project soonish anyway.